datafusion/expr/src/expr.rs

```diff
@@ -2607,11 +2793,23 @@ pub(crate) fn schema_name_from_exprs_comma_separated_without_space(
     schema_name_from_exprs_inner(exprs, ",")
 }
 
+/// Get `sql_name` for Vector of expressions.
+pub(crate) fn sql_name_from_exprs_comma_separated_without_space(
```

Comment:
`without_space` is the old format and ideally we should standardize the format #10364 so better not to introduce another format without good reason